data/learning.yml

```diff
@@ -64,6 +64,13 @@ books:
       - link: /learn/best_practices/callbacks
       - link: /learn/best_practices/type_casting
 
+  - title: Python Fortran Rosetta Stone
```

Comment:
```suggestion
  - title: Python--Fortran Rosetta Stone
```
I think it makes sense to have an en dash in the title there.

Reply:
Might have to enable extension for it work with `--` though.

https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#typography

Reply:
@henilp105 did it render as en dash without extension? just curious

Reply:
@zmoon yes :smile: , it seems that markdown would be supporting this.